src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1287,10 +1292,17 @@ private String lastDottedPart(String input) {
     /** Whether the symbol we are walking was defined in an extern file */
     private final boolean isExtern;
 
-    private TreeWalker(JSTypeRegistry typeRegistry, Set<String> provides, boolean isExtern) {
+    private final boolean isGoogNamespace;
+
+    private TreeWalker(
+        JSTypeRegistry typeRegistry,
+        Set<String> provides,
+        boolean isExtern,
+        boolean isGoogNamespace) {
```

Comment:
We need one location in the code to explain what's special about goog base. Maybe this is that place.

Reply:
Added a comment - does it work?

Reply:
looks good